qa/workunits/rbd/qemu-iotests.sh

```diff
@@ -5,15 +5,18 @@
 # require the admin ceph user, as there's no way to pass the ceph user
 # to qemu-iotests currently.
 
-# This will only work with particular qemu versions, like 1.0. Later
-# versions of qemu include qemu-iotests directly in the qemu
-# repository.
 testlist='001 002 003 004 005 008 009 010 011 021 025 032 033 055 077'
 
 git clone https://github.com/qemu/qemu.git
-# use v2.2.0-rc3 (last released version that handles all the tests
 cd qemu
-git checkout 2528043f1f299e0e88cb026f1ca7c40bbb4e1f80
+if `lsb_release -da | grep -i xenial > /dev/null`; then
```

Comment:
@dillaman You could use grep -q option to avoid /dev/null redirect. Or just `if [ "$(lsb_release -sc)" = xenial ]`.

Also, do you expect this to work for post xenial releases? Then might be `"$(lsb_release -is)" = Ubuntu -a "$(lsb_release -rs)" '>' 16`?